terraform/scenarios/omnibus-external-openldap/files/chef-server.rb

```diff
@@ -0,0 +1,16 @@
+opscode_erchef['keygen_start_size'] = 30
+
+opscode_erchef['keygen_cache_size'] = 60
+
+nginx['ssl_dhparam'] = '/etc/opscode/dhparam.pem'
+
+data_collector['token'] = 'foobar'
+
+profiles['root_url'] = 'http://localhost:9998'
+
+ldap['base_dn'] = 'ou=chefs,dc=chef-server,dc=dev'
+ldap['bind_dn'] = 'cn=admin,dc=chef-server,dc=dev'
+ldap['bind_password'] = 'H0\/\/!|\/|3tY0ur|\/|0th3r'
+ldap['host'] = 'ldap.chef-server.dev'
+ldap['login_attribute'] = 'uid'
+ldap['tls_enabled'] = true
```

Comment:
it might be worth making a note what happens / does not happen if ldap['ssl_enabled'] is set to true.

Reply:
Ok, I'll toss a comment in the code.

I'm still confused by what the original intent of `ldap['ssl_enabled'] = true` was and whether it was only intended for Active Directory / Global Catalog.

Whenever I've set `ldap['ssl_enabled'] = true` against an OpenLDAP instance I only see the following error on the Chef Manage login screen:
```
The AD/LDAP server is down. Please contact an administrator. 
```

A TCP dump captured during the login attempt shows the connection being reset:
![TCP dump](https://user-images.githubusercontent.com/770940/68395151-774dd600-012c-11ea-9858-1235abe191f7.png)

With `ldap['ssl_enabled'] = false` and `ldap['tls_enabled'] = true` we are able to connect to the OpenLDAP instance without issue:
![tcpdump](https://user-images.githubusercontent.com/770940/68395431-ef1c0080-012c-11ea-96c5-2cf0d96604f5.png)